src/components/ConsoleLogComponent.tsx

```diff
@@ -0,0 +1,25 @@
+import config from './.config'; // Relative path to your File
+
+/**
+ /
```

Comment:
delete this line

Reply:
I need that line for reading the config variable